modules/op-release-notes-1-8.adoc

```diff
@@ -26,14 +26,20 @@ include::snippets/technology-preview.adoc[]
 
 * Support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
 +
-This change affects the pipeline component, which includes the `TaskRun`, `PipelineRun`, `Task`, `Pipeline`, and similar `tekton.dev/v1alpha1` resources. The `operator.tekton.dev/v1alpha1` API version is not deprecated and still used by the Operator.
+This change affects the pipeline component, which includes the `TaskRun`, `PipelineRun`, `Task`, `Pipeline`, and similar `tekton.dev/v1alpha1` resources. Rather, update existing resources to use `apiVersion: tekton.dev/v1beta1` as described in link:https://tekton.dev/docs/pipelines/migrating-v1alpha1-to-v1beta1/[Migrating From Tekton v1alpha1 to Tekton v1beta1]
```

Comment:
```suggestion
This change affects the pipeline component, which includes the `TaskRun`, `PipelineRun`, `Task`, `Pipeline`, and similar `tekton.dev/v1alpha1` resources. As an alternative, update existing resources to use `apiVersion: tekton.dev/v1beta1` as described in link:https://tekton.dev/docs/pipelines/migrating-v1alpha1-to-v1beta1/[Migrating From Tekton v1alpha1 to Tekton v1beta1]
```
To my ears, `Rather` doesn't sound quite right here to indicate a workaround/alternative. It seems like it should have a parallel contrasting sentence before it like "don't do this. Rather, do this..."

Reply:
Updated. Thanks.